python/tests/conftest.py

```diff
@@ -1,10 +1,57 @@
 # Copyright (c) Microsoft. All rights reserved.
 
 import os
+import typing as t
+import warnings
 
 import pytest
 
 import semantic_kernel as sk
+from semantic_kernel.memory.null_memory import NullMemory
+from semantic_kernel.orchestration.context_variables import ContextVariables
+from semantic_kernel.orchestration.sk_context import SKContext
+from semantic_kernel.orchestration.sk_function import SKFunction
+from semantic_kernel.skill_definition.read_only_skill_collection import (
+    ReadOnlySkillCollection,
+)
+
+
+@pytest.fixture(autouse=True)
+def enable_debug_mode():
+    """Set `autouse=True` to enable easy debugging for tests.
+
+    How to debug:
+    1. Ensure [snoop](https://github.com/alexmojaki/snoop) is installed
```

Comment:
why not put this in pyproject.toml?

Reply:
I didn't know if you wanted the additional dependency. I could add it to pyproject.toml, but you would still have to update the poetry lock file yourself because I don't have permissions to push changes to master. If it helps, I can put up a separate PR with the poetry update.